app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt

```diff
@@ -13,8 +13,9 @@ import org.mozilla.fenix.session.PerformanceActivityLifecycleCallbacks
  * An application class which knows how to migrate Fennec data.
  */
 class MigratingFenixApplication : FenixApplication() {
-
     init {
+        recordOnInit() // DO NOT MOVE ANYTHING ABOVE HERE: the timing of this measurement is critical.
```

Comment:
Since `MigratingFenixApplication` extends `FenixApplication` both initializer blocks should get invoked resulting in `recordOnInit()` getting called twice.

Reply:
That's by design: `StartupFrameworkStartMeasurement` handles multiple callers. I'll add a comment to `StartupTimeline.onApplicationInit` to make that assumption clearer in that wrapper method though.